nginx/service.conf

```diff
@@ -0,0 +1,26 @@
+upstream 127.0.0.1{
```

Comment:
1) It's better to use upstream with aliases and not with IP addreses.
2) The main purpose of `upstream` is to describe a **group** of servers, not a single server. So I think it's redundant.